DexcomCGMKitUI/ViewModels/ReplacementWorkflowViewModel.swift

```diff
@@ -59,4 +59,31 @@ class ReplacementWorkflowViewModel {
         }
         return true
     }
+    
+    func validTransmitterID(_ rawTransmitterID: String) -> Bool {
+        let transmitterID = DexcomTransmitterID(rawTransmitterID)
+        return validTransmitterID(transmitterID)
+    }
+    
+    func validTransmitterID(_ transmitter2DBarcode: Dexcom2DBarcode?) -> Bool {
+        var transmitterID: DexcomTransmitterID?
+        if let transmitter2DBarcode = transmitter2DBarcode,
+            transmitter2DBarcode.isValid(),
+            let transmitterIDString = transmitter2DBarcode.code
+        {
+            transmitterID = DexcomTransmitterID(transmitterIDString)
+        }
+        return validTransmitterID(transmitterID)
+    }
+    
+    func validTransmitterID(_ transmitterID: DexcomTransmitterID?) -> Bool {
+        guard transmitterID != nil else {
+            return false
+        }
+        
+        workflowCompletionHandler = { [weak self] in
+            self?.replaceTransmitterCompletionHandler!(transmitterID!)
+        }
+        return true
```

Comment:
Avoid force-unwrapping.  Consider: 
```suggestion
        guard let transmitterID = transmitterID else {
            return false
        }
        
        workflowCompletionHandler = { [weak self] in
            self?.replaceTransmitterCompletionHandler?(transmitterID)
        }
        return true
```